src/s2i/bin/usage

```diff
@@ -7,7 +7,7 @@ if [ ${PLATFORM} == "el7" ]; then
 elif [ ${PLATFORM} == "el8" ]; then
```

Comment:
This is not specific for this PR for adding Fedora, but there is no branch for el9 here.

Reply:
I will fix it with this pull request. Thanks for catching it.

Reply:
Fixed by https://github.com/sclorg/varnish-container/pull/115/commits/aea3391316055858a2e5b9895820baa0b4c827e6